db/migrate/20180115214144_add_announcement_to_organization_contents.rb

```diff
@@ -1,5 +1,6 @@
 class AddAnnouncementToOrganizationContents < ActiveRecord::Migration
   def change
     add_column :organization_contents, :announcement, :string, default: ""
+    Medium.where(type: "org_attached_image").update_all(type: "organization_attached_image")
```

Comment:
Hasn't this migration already run on staging? so this change won't do anything...

Reply:
Has it? This PR hasn't merged yet, why would it have deployed to staging?

Reply:
yeah, but a whole extra migration just for this seems like overkill. I'll fix it on the console.

Reply:
👍